src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -1993,6 +1997,12 @@ def test_aks_nodepool_snapshot(self, resource_group, resource_group_location):
         })
         print("The snapshot resource id %s " % snapshot_resource_id)
 
+        # update tags on nodepool snapshot
```

Comment:
Queued [live test pipeline](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=75629391&view=results) to validate the change.

Reply:
Test passed.